gradle.properties

```diff
@@ -2,20 +2,20 @@
 org.gradle.jvmargs=-Xmx1G
 
 # Fabric Properties
-	# check these on https://fabricmc.net/versions.html
-	minecraft_base_version=1.19
-	minecraft_version=1.19
-	yarn_mappings=1.19+build.1
-	loader_version=0.14.7
+	# check these on https://fabricmc.net/develop/
+	minecraft_base_version=1.19.3
+	minecraft_version=1.19.3
+	yarn_mappings=1.19.3+build.3
+	loader_version=0.14.11
 
 # Mod Properties
-	mod_version = 0.1.7
+	mod_version = 0.2.0
```

Comment:
Hey, incredibly sorry for not looking at this for so long - I've been quite busy lately and also frankly just forgot. Looking over this PR again, I'd say it's ready for merging as soon as you revert this version bump. Nothing about the mod actually functionally changed, thus there's no need to bump the mod's own version - the `minecraft_base_version` part is used to signal the version change

Looking forward to getting this updated!